Makefile

```diff
@@ -17,7 +17,7 @@
 CONTAINER_CMD?=docker
 
 CSI_IMAGE_NAME=$(if $(ENV_CSI_IMAGE_NAME),$(ENV_CSI_IMAGE_NAME),quay.io/cephcsi/cephcsi)
-CSI_IMAGE_VERSION=$(if $(ENV_CSI_IMAGE_VERSION),$(ENV_CSI_IMAGE_VERSION),canary)
```

Comment:
Is the intention to change this back to 1.2-canary post the release? I am asking, as post 1.2.0 if any PR is merged to the 1.2.0 branch, we would update the same image in quay (afaict), which is not desirable.

Reply:
yes will revert it back to 1.2-canary